### ISSUE TITLE
Multiple Password in Toolbar, No Selection in Login/Password Field

### ISSUE THREAD

**Issue by wjwieland** [nextcloud/passman-webextension#256]
Originally opened in Nextcloud/Passman(#477), then re-opened here.
### Steps to reproduce

1. Similar entries in DB. For example: https://me.myserver.org/nextcloud/login.php and https://me.meserver.org/blog/
2. Open either link
3. The passman icon in the toolbar will display entries for both sites, but there will be now icons in the username or password fields from which to choose a password.
4. One must log in manually.

### Expected behaviour
A. Preferable - DB should look at more than just the domain name so that if there are two logins on the same site with different paths the correct password should be entered.

B. Regardless of whether or not more than one password is available or not, the login and password fields should contain the icon and a list of the potential entries.
### Actual behaviour
Tell us what happens instead
Please see steps to reproduce.
### Configuration
**Operating system**: 
Linux - Ubuntu 17.10.1 (server)
Any - (Client)
**Browser**: 
Chromium, Google Chrome
**Extensions that might cause interference**: 
Unknown
**Passman version**: 
2.1.4
**Extension version**: 
2.1.1
**Nextcloud version**: 
   13.02
#### Browser log
<details>
<summary>Browser log</summary>
```
[wjwieland.dvrdns.org-1526236212978.log](https://github.com/nextcloud/passman-webextension/files/1998802/wjwieland.dvrdns.org-1526236212978.log)

```
</details>   
Please do let me know if anything else I might provide would help.  For example, are there settings somewhere that I am missing? 
Thank you

**Comment by violoncelloch** [comment]
continuing discussion from nextcloud/passman#477
you can find the advanced settings inside the passman extension > settings > advanced settings (expandable, right above the save button) there you can adjust these settings regarding the detection of the url...
also there is the debug setting I mentioned... so activating the debug for Passman inside nextcloud only activated the debug mode for the webapplication inside nextcloud, not the extension, and though only affects the webapp... could you retry activating debug mode in the extension?

**Comment by wjwieland** [comment]
Aha!  Sorry it took so long to get back to this issue.  
What I have found is that the settings for the extension on a window box running google chrome Version 66.0.3359.139 (Official Build) (64-bit) can not show the advanced settings.  The settings window does not show a scroll bar on the right, leaving only three settings available.
![passman_win10](https://user-images.githubusercontent.com/5116738/40009500-d3bdbc22-5767-11e8-99e5-cb4232b89d5f.JPG)

However on a linux box (sparky in this case) and using chromium build 62.0.3202.89(Developer Build) the  advanced settings does result in a scrollable window in which I can adjust all settings.  (Is this a new issue?)
![passman_linux](https://user-images.githubusercontent.com/5116738/40009642-4357804a-5768-11e8-8ae5-6f5ab5483dbf.JPG)

That settings window issue is pretty important I think.  When I changed the settings on the linux box with chromium, passman started working as expected, with only one credential in the list and the form auto-filled correctly.  

Please let me know if there is anything else you would like me to provide. 
Thanks for you patience!

**Comment by violoncelloch** [comment]
hmm, hard to say for me if this is an issue with the passman extension or possibly an issue with your chrome
Maybe @brantje has an idea?

**Comment by wjwieland** [comment]
Interesting... I got home (was away when I opened this issue) and got back on my normal home Linux laptop and found that Google-Chrome does NOT show all the advanced settings.  However Chromium-browser does show them all.  This seems to be a browser issue more than anything.  Both chromium-browser and google-chrome are at their latest stable release on my home laptop.

I do not know if chromium-browser works on win-10, as I do not have the time to compile it for that platform and I don't really trust the installers that are available.  (Wiindows is bad enough without risking whatever might be delivered along with some install from some untrusted source, and I don't want to have to spend hours cleaning up whatever mess might be left, Window is just not worth my time).

At any rate, and to summarize my findings:  The web extension for passman does not offer up the advanced settings on google-chrome.  It does offer them up on chromium-browser.  Hope this is helpful.
